client/app/components/pas-form.js

```diff
@@ -4,10 +4,15 @@ import { tracked } from '@glimmer/tracking';
 
 export default class PasFormComponent extends Component {
   @tracked dcpUrbanrenewalarea;
+
   @tracked dcpLanduseactiontype2;
+
   @tracked dcpProjectareaindustrialbusinesszone;
+
   @tracked dcpIsprojectarealandmark;
+
   @tracked dcpIsinclusionaryhousingdesignatedarea;
+
   @tracked dcpDiscressionaryfundingforffordablehousing;
```

Comment:
These are properties that'll be on the pas-form model. I would just pass in a dummy object for now.

Reply:
Happy to help with this any time